### PR TITLE
Spotless eclipse formatter apply

### PR DESCRIPTION
**Pull request by rg911**

### CODE REVIEW

File: src/main/java/org/nem/core/crypto/ed25519/arithmetic/Ed25519EncodedFieldElement.java [hunk 1]
```diff
@@ -20,13 +19,13 @@
 	 */
 	public Ed25519EncodedFieldElement(final byte[] values) {
 		switch (values.length) {
-			case 32:
+			case 32 :
 				this.zero = Ed25519Field.ZERO_SHORT;
 				break;
-			case 64:
+			case 64 :
```
Comment (rg911) [inline src/main/java/org/nem/core/crypto/ed25519/arithmetic/Ed25519EncodedFieldElement.java:25]: @Jaguar0625 not sure if this (whitespace) is alright with you?

**Review by rg911 — COMMENTED** [review]
looks good in general, just not sure if the switch case should have whitespace or not?

File: src/main/java/org/nem/core/crypto/ed25519/arithmetic/Ed25519EncodedFieldElement.java (hunk 1 above)
Comment (Jaguar0625) [reply src/main/java/org/nem/core/crypto/ed25519/arithmetic/Ed25519EncodedFieldElement.java]: for some reason i couldn't find the setting for this yesterday, but found it today, so updated 👍

File: src/main/java/org/nem/core/crypto/ed25519/arithmetic/Ed25519FieldElement.java [hunk 2]
```diff
@@ -46,8 +46,8 @@ public boolean isNonZero() {
 	}
 
 	/**
-	 * Adds the given field element to this and returns the result.
-	 * <b>h = this + g</b>
+	 * Adds the given field element to this and returns the result. <b>h = this + g</b>
+	 *
```
Comment (gimre-xymcity) [inline src/main/java/org/nem/core/crypto/ed25519/arithmetic/Ed25519FieldElement.java:50]: br? (and in few more below)

File: src/main/java/org/nem/core/crypto/ed25519/arithmetic/Ed25519FieldElement.java [hunk 3]
```diff
@@ -438,15 +432,16 @@ public Ed25519FieldElement squareAndDouble() {
 	}
 
 	/**
-	 * Squares this field element, optionally multiplies by two and returns the result.
-	 * <b>h = 2 * this * this</b> if dbl is true or
-	 * <b>h = this * this</b> if dbl is false.
+	 * Squares this field element, optionally multiplies by two and returns the result. <b>h = 2 * this * this</b> if dbl is true or <b>h =
+	 * this * this</b> if dbl is false.
+	 *
```
Comment (gimre-xymcity) [inline src/main/java/org/nem/core/crypto/ed25519/arithmetic/Ed25519FieldElement.java:437]: messy here

File: src/test/java/org/nem/core/crypto/ed25519/arithmetic/MathUtils.java [hunk 4]
```diff
@@ -12,15 +12,7 @@
...
-			0,
-			26,
-			26 + 25,
-			2 * 26 + 25,
-			2 * 26 + 2 * 25,
-			3 * 26 + 2 * 25,
-			3 * 26 + 3 * 25,
-			4 * 26 + 3 * 25,
-			4 * 26 + 4 * 25,
+			0, 26, 26 + 25, 2 * 26 + 25, 2 * 26 + 2 * 25, 3 * 26 + 2 * 25, 3 * 26 + 3 * 25, 4 * 26 + 3 * 25, 4 * 26 + 4 * 25,
```
Comment (gimre-xymcity) [inline src/test/java/org/nem/core/crypto/ed25519/arithmetic/MathUtils.java:15]: ugh 😬

**Review by gimre-xymcity — APPROVED** [review]

File: src/main/java/org/nem/core/crypto/ed25519/arithmetic/Ed25519FieldElement.java (hunk 2 above)
Comment (Jaguar0625) [reply src/main/java/org/nem/core/crypto/ed25519/arithmetic/Ed25519FieldElement.java:50]: nope, that is from original. don't know why, but didn't change.

File: src/main/java/org/nem/core/crypto/ed25519/arithmetic/Ed25519FieldElement.java (hunk 3 above)
Comment (Jaguar0625) [reply src/main/java/org/nem/core/crypto/ed25519/arithmetic/Ed25519FieldElement.java:437]: it reads ok to me. i can add <br> if you'd prefer to make it two lines.

File: src/test/java/org/nem/core/crypto/ed25519/arithmetic/MathUtils.java (hunk 4 above)
Comment (Jaguar0625) [reply src/test/java/org/nem/core/crypto/ed25519/arithmetic/MathUtils.java:15]: not much we can do about that :/

File: src/main/java/org/nem/core/crypto/ed25519/arithmetic/Ed25519FieldElement.java (hunk 3 above)
Comment (gimre-xymcity) [reply src/main/java/org/nem/core/crypto/ed25519/arithmetic/Ed25519FieldElement.java:437]: I think would look cleaner if both
... if dbl is true
... if dbl is false

would be on separate lines, but no strong opinion (also not really important I guess)

**Pull request merged**